content/node/knex/_index.md

```diff
@@ -93,37 +93,51 @@ In the database server logs, the comment's fields are:
-`route`|`<the route used>`|The URL-quoted route used to match the express.js controller|`route='%5Epolls/%24'`
-`traceparent`|`<XX-TRACE_ID-SPAN_ID-TRACE_OPTIONS>`|The serialized [W3C TraceContext.Traceparent](https://www.w3.org/TR/trace-context/#traceparent-field)|`traceparent='00-f5e3fa7fb15a461dbf3b03690e4bd5e1-e6de66630cd19b9a-01'`
-`tracestate`|`<KEY1=VALUE1,KEY2=VALUE2,...>`|The serialized [W3C TraceContext.Tracestate](https://www.w3.org/TR/trace-context/#tracestate-field)|`tracestate='congo%3Dt61rcWkgMzE%2Crojo%3D00f067aa0ba902b7'`
+| Field             | Format                        | Description                                                                                          | Example                                                                 |
+|-------------------|-------------------------------|------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------|
+| `db_driver`       | `<database_driver>:<version>` | URL quoted name and version of the database driver                                                   | `db_driver='knex%3A0.16.5'`                                             |
+| `route`           | `<the route used>`            | URL quoted route used to match the express.js controller                                             | `route='%5E%2Fpolls%2F`                                                 |
+| `traceparent`     | `<traceparent header>`        | URL quoted [W3C `traceparent` header](https://www.w3.org/TR/trace-context/#traceparent-header)       | `traceparent='00-3e2914ebce6af09508dd1ff1128493a8-81d09ab4d8cde7cf-01'` |
+| `tracestate`      | `<tracestate header>`         | URL quoted [W3C `tracestate` header](https://www.w3.org/TR/trace-context/#tracestate-header)         | `tracestate='rojo%253D00f067aa0ba902b7%2Ccongo%253Dt61rcWkgMzE'`        |
 
 #### Options
 
-When creating the middleware, one can optionally toggle attributes to be set in the comments by passing in the option `include` which is a map
+When creating the middleware, one can optionally configure the injected
+comments by passing in the `include` and `options` objects:
 
 ```javascript
-wrapMainKnexAsMiddleware(Knex, include={...});
+wrapMainKnexAsMiddleware(Knex, include={...}, options={...});
 ```
 
-Field|On by default
----|---
-route|<div style="text-align: center">&#10004;</div>
-tracestate|<div style="text-align: center">&#10060;</div>
-traceparent|<div style="text-align: center">&#10060;</div>
-db_driver|<div style="text-align: center">&#10060;</div>
+##### `include`
+A map of values to be optionally included in the SQL comments.
+
+| Field           | On by default |
+| --------------- | ------------- |
+| db_driver       | No            |
+| route           | Yes           |
+| traceparent     | No            |
+| tracestate      | No            |
```

Comment:
Please revert the checkmark and x as they are more visually indicative (I found) than Yes or No.